fhi_users/auth/auth_forms.py

```
@@ -53,3 +53,40 @@ class ChangePasswordForm(forms.Form):
+
+        return cleaned_data
+
+    def clean_email(self):
+        email = self.cleaned_data.get("email")
+        from django.contrib.auth import get_user_model
+
+        User = get_user_model()
+        if User.objects.filter(email=email).exists():
+            raise forms.ValidationError(
+                "An account with this email already exists. Please log in instead."
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)